lib/heartcheck/services/response_formatter.rb

```diff
@@ -0,0 +1,30 @@
+module Heartcheck
+  module Services
+    class ResponseFormatter
+      attr_reader :response_formatter
+
+      def initialize(response_formatter)
+        @response_formatter = response_formatter
```

Comment:
If you agree with my suggestion in https://github.com/locaweb/heartcheck/pull/33/files#r432548255 , I think it is good to use the same name in this variable `response_formatter`

Reply:
Done!